functions

```diff
@@ -500,19 +502,19 @@ add_dir() {
     #   $1: pathname on initcpio
     #   $2: mode (optional)
 
-    if [[ -z $1 || $1 != /?* ]]; then
+    if [[ -z "$1" || "$1" != /?* ]]; then
         return 1
     fi
 
-    local path=$1 mode=${2:-755}
+    local path="$1" mode="${2:-755}"
 
-    if [[ -d $BUILDROOT$1 ]]; then
+    if [[ -d "${BUILDROOT}${1}" ]]; then
         # ignore dir already exists
         return 0
     fi
 
     quiet "adding dir: %s" "$path"
-    command install -dm$mode "$BUILDROOT$path"
+    command install -dm"${mode}" "${BUILDROOT}${path}"
```

Comment:
Inconsistent use of braces. Either add it to all the variables or don't add it anywhere. Adding it in a few places makes the style inconsistent.

Reply:
Most style guides push for consistent use, one way or another, since consistent code is good code.

In this particular line, they aren't needed anywhere, which is how the code was working prior to this change.

Reply:
It just feel dirty to me to place variables inline without enclosing them.